base_multi_tag/__openerp__.py

```diff
@@ -0,0 +1,28 @@
+{
+    "name": "Tags",
+    "version": "0.0.3",
+    "author": "Management and Accounting On-line",
+    "website": "",
+    "summary": "Tags",
+    'category': 'Hidden/Dependency',
+    "depends": [
+        "base",
+        "base_action_rule",
+    ],
+    "description": """
+        Provides generic tags logic and basic views.
+        Integration with other modules should be implemented as sparate modules
```

Comment:
This probably too abstract.
Could you please provide a README with a better description?